Extensions/TileMap/tilemapruntimeobject-pixi-renderer.ts

```diff
@@ -148,11 +148,11 @@ namespace gdjs {
     }
 
     getWidth(): float {
-      return this._pixiTileMapData?.tilemapWidth;
+      return this._pixiTileMapData ? this._pixiTileMapData.tilemapWidth : this._pixiObject.width;
```

Comment:
`this._pixiTileMapData ? this._pixiTileMapData.tilemapWidth : 0`

like everywhere else. If your _pixiTileMapData is null, this means that your tilemap is not loaded.

Reply:
My bad this was a mistake on my side. Should be:

`this._pixiTileMapData ? this._pixiTileMapData.tilemapWidth * this._pixiObject.scale.x : 0 `

You take the original width and multiply by the scale factor :) This should give you the proper size, as long as tilemapWidth is the proper original width.

Note that here it is the *actual width* that you need to return.
In the IDE, in `getDefaultWidth`, you return the default width.